prover/witness_generator/src/basic_circuits.rs

```diff
@@ -15,7 +15,7 @@ use circuit_definitions::{
     encodings::recursion_request::RecursionQueueSimulator,
     zkevm_circuits::fsm_input_output::ClosedFormInputCompactFormWitness,
 };
-use multivm::vm_latest::{
+use multivm::vm_1_4_2::{
```

Comment:
If I understand correctly making witness generator work with 1.5.0 is out of scope for this PR. nit: please add a TODO